sql/migrations/reports/000006_alter_reports_auto_vaccum.up.tmpl

```diff
@@ -0,0 +1 @@
+ALTER TABLE reports SET (autovacuum_vacuum_cost_limit = {{ .AutoVacuumCostLimit }});
```

Comment:
Will this migration run if we change the configuration? It is better to run this query after migration has been completed to ensure it always going to apply when the server starts.

Reply:
Running the query after migration now.